src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,123 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int CAPACITY = 10;
+    private StackTraceElement[] stackTraceElements = Thread.currentThread().getStackTrace();
+    private final int indexMethod = stackTraceElements.length - 2;
+    private StackTraceElement element = stackTraceElements[indexMethod];
```

Comment:
why are you need this?

Reply:
I wanted to display in which method the exception had appeared